src/config/routes/authenticated.js

```diff
@@ -0,0 +1,6 @@
+const mount = require('koa-mount');
+const userResource = require('resources/user/authenticated');
```

Comment:
I would name file `index.js` and let others decide if resource require authentication.